### ISSUE TITLE
all images change into strange text

### ISSUE THREAD

**Issue by Ntlneo** [microsoft/vscode#179568]
Type: <b>Bug</b>

I just click on any file image such as .png and bug happens. It also occurs with images in extension

VS Code version: Code 1.77.1 (b7886d7461186a5eac768481578c1d7ca80e2d21, 2023-04-04T23:21:11.906Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6600K CPU @ 3.50GHz (4 x 3504)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.96GB (7.75GB free)|
|Process Argv|--crash-reporter-id 0f7c0729-8763-4b4a-8675-0fe6d4b3a4a3|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
vs-chatgpt|0xf|1.5.0
vscode-django|bat|1.10.0
vscode-mysql-client2|cwe|6.3.5
python-environment-manager|don|1.0.4
python-extension-pack|don|1.7.0
vscode-mysql|for|0.4.1
java-test|jav|0.0.2
vsc-python-indent|Kev|1.18.0
vscode-gutter-preview|kis|0.30.0
gitignore|mic|1.0.1
isort|ms-|2022.8.0
python|ms-|2023.6.0
vscode-pylance|ms-|2023.4.10
jupyter|ms-|2023.3.1000892223
jupyter-keymap|ms-|1.1.0
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
js-debug-nightly|ms-|2023.4.517
vscode-typescript-next|ms-|5.1.20230408
autodocstring|njp|0.6.1
fabric8-analytics|red|0.3.6
java|red|1.16.0
vscode-xml|red|0.24.0
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.49.1
vscode-java-dependency|vsc|0.21.2
vscode-java-pack|vsc|0.25.10
vscode-java-test|vsc|0.38.2
vscode-maven|vsc|0.41.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30610678
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
showlangstatbar:30672706
vsccsb:30705552
defaultazurewalk:30687958
pythonms35:30701012
pythonclientmv:30708685

```

</details>

**Comment by Ntlneo** [comment]
here's the bug attachments
![Screenshot_30](https://user-images.githubusercontent.com/32924628/230783277-e66b6c81-9680-4542-bb89-630a67496e46.png)
![Screenshot_29](https://user-images.githubusercontent.com/32924628/230783280-795744e2-2021-4265-b783-49da477feb65.png)

**Comment by gjsjohnmurray** [comment]
Please use the Start Extension Bisect command to investigate which extension is causing the problem.

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by Ntlneo** [comment]
> Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

I don't know. My VScode was the lastest version already. I tried a few ways but bug still occured:
1. uninstall some extension related image such as preview image...
2. Turn on/off some options in VScode setting by sreach 'image' 
3. Delete projects, delete workspaces, then import them again.
4. Turn on/off venv

So finally I reinstalled the VScode into a new folder. Then the bug disappeared miraculously.
Now my VScode with all extensions and old vevn works normally.
